packages/fields/src/fields/featured-image/featured-image-view.tsx

```diff
@@ -0,0 +1,38 @@
+/**
+ * WordPress dependencies
+ */
+import { useSelect } from '@wordpress/data';
+import { store as coreStore } from '@wordpress/core-data';
+
+/**
+ * Internal dependencies
+ */
+import type { BasePost } from '../../types';
+import type { DataViewRenderFieldProps } from '@wordpress/dataviews';
+
+export const FeaturedImageView = ( {
+	item,
+}: DataViewRenderFieldProps< BasePost > ) => {
```

Comment:
It seems this could be implemented in a way that is not specific to "featured media", but rather to a `media` type. For example, what if we do something along these lines:

```js
FeaturedImageView = ({
  item,
}) {
  const url = item.getValue();
  return (<img src={ url } );
}
```

If this was a `media` type field, the field "featured media" could be declared as:

```js
{
  id: 'featured-media',
  type: 'media',
  getValue: () => { /* convert item to URL here */ },
}
```